app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -111,22 +111,6 @@ def provided_attributes
       super.merge({:mac => :mac})
     end
 
-    def use_v4=(value)
-      value = case value
-              when true, '1'
-                true
-              else
-                false
-              end
-      attrs[:ovirt_use_v4] = value
```

Comment:
Do we need a migration to clean up this attribute from existing CRs?

Reply:
yes I will add it.